python/script/ci-test

```diff
@@ -2,6 +2,8 @@
 
 set -e
 
-bundle install
+pyenv exec pip --disable-pip-version-check install --use-pep517 -r "dev-requirements.txt"
 pyenv exec flake8 helpers/. --count --exclude=./.*,./python/spec/fixtures --show-source --statistics
+
+bundle install
```

Comment:
This move is unrelated and just aesthetic cleanup, right?

Reply:
I reverted to the previous order 👍.